README.md

```diff
@@ -103,6 +103,224 @@ PSPDFKit.present('document.pdf', {
 - Install npm packages: `npm install` in `samples/Catalog` directory.
 - Run the app with `react-native-cli`: `react-native run-ios`
 
+### Android
+
+#### Requirements
+
+- Android SDK
+- PSPDFKit 6
+- react-native >= 0.34.1
+
+#### Getting Started
+
+Let's create a simple app that integrates `pspdfkit-*.aar` and uses the react-native-pspdfkit module.
+
+1. Make sure `react-native-cli` is installed: `npm install -g react-native-cli`
+2. Create the app with `react-native init YourApp`.
+3. Step into your newly created app folder: `cd YourApp`.
+4. Install `react-native-pspdfkit` from GitHub: `react-native install github:PSPDFKit/react-native`.
+5. Add dependencies to `YourApp/node_modules/react-native-pspdfkit/build.gradle`.
```

Comment:
Yes, `node_modules` is a transient folder, and yes is excluded from being checked into a repo. This folder is generated by react installation process `react-native install`.